env0/resource_template.go

```diff
@@ -243,7 +243,6 @@ func resourceTemplateRead(ctx context.Context, d *schema.ResourceData, meta inte
 	d.Set("description", template.Description)
 	d.Set("github_installation_id", template.GithubInstallationId)
 	d.Set("token_id", template.TokenId)
-	d.Set("gitlab_project_id", template.GitlabProjectId)
```

Comment:
Can we cover this with some test?
It feels like we aren't asserting that the value is set after create/update.

Reply:
3fbecee

I am not sure if I did it correctly or if I missed something on mocks. please take a look @yaronya

Reply:
fixed 7ae0d91